pkg/scheduler/framework/runtime/framework.go

```diff
@@ -347,13 +343,13 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, opts ...Opti
 	}
 
 	if len(f.queueSortPlugins) == 0 {
-		return nil, fmt.Errorf("no queue sort plugin is enabled")
+		return nil, fmt.Errorf("no queue sort plugin is enabled for profile with scheduler name %q", profile.SchedulerName)
```

Comment:
why remove the scheduler name from all of these checks?

Reply:
ah, scheduler name is added here.